createdisk.sh

```diff
@@ -27,6 +27,9 @@ ${SSH} core@${VM_IP} -- 'sudo find /var/log/ -iname "*.log" -exec rm -f {} \;'
 
 install_packages ${VM_IP}
 
+# Enable cockpit socket
+${SSH} core@${VM_IP} -- 'sudo systemctl enable cockpit.socket'
```

Comment:
I'd keep this in `install_packages`, `prepare_hyperV` is already doing more than just package installation. And this commit can be squashed in the previous one. The comment is also not really needed, it's easy enough to just read the shell code.